src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-501/rgw-501.component.html

```diff
@@ -0,0 +1,12 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li i18n
+        class="breadcrumb-item">Object Gateway</li>
+  </ol>
+</nav>
+<cd-info-panel>
+  {{ message }}
+  <ng-container i18n>
+    Please consult the <a href="http://docs.ceph.com/docs/mimic/mgr/dashboard/#enabling-the-object-gateway-management-frontend">documentation</a> about how to enable RGW for the dashboard.
```

Comment:
Small textual improvement suggestion:

_Please consult the documentation on how to configure and enable the object gateway management functionality._